vaadin-theme-property-mixin.html

```diff
@@ -13,7 +13,18 @@
       return {
         /**
          * Helper property with theme attribute value facilitating propagation
-         * in shadow DOM. Allows using `theme$="[[theme]]"` in the template.
+         * in shadow DOM.
+         *
+         * Enables the component implementation to propagate the `theme`
+         * attribute value to the subcomponents in Shadow DOM by binding
+         * the subcomponent’s "theme" attribute to the `theme` property of
+         * the host.
+         *
+         * **NOTE:** Extending the mixin only provides the property for binding,
+         * and does not make the propagation alone.
+         *
+         * See [Theme Attribute and Subcomponents](https://github.com/vaadin/vaadin-themable-mixin/wiki/5.-Theme-Attribute-and-Subcomponents).
```

Comment:
Can you also append the content from the linked Wiki page to README?

Reply:
Done.